pkg/verifier/aws/entry_point.go

```diff
@@ -61,6 +61,23 @@ func (a *AwsVerifier) ValidateEgress(vei verifier.ValidateEgressInput) *output.O
 		return a.Output.AddError(err) // fatal
 	}
 
+	cleanupSecurityGroup := false
+	if vei.AWS.SecurityGroupId == "" {
+		vpcId, err := a.GetVpcIdFromSubnetId(vei.Ctx, vei.SubnetID)
+		if err != nil {
+			return a.Output.AddError(err)
+		}
+
+		createSecurityGroupOutput, err := a.CreateSecurityGroup(vei.Ctx, vei.Tags, "ONV_security_group", vpcId)
```

Comment:
Can the name passed in here please match the name of the EC2 instance as well?

Reply:
Added it as part of the latest PR